apps/extension/src/routes/popup/enter-password.tsx

```diff
@@ -3,36 +3,87 @@ import { useState } from 'react';
 import { PopupPath } from './paths';
 import { usePopupNav } from '../../utils/navigate';
 import { passwordSelector } from '../../state/password';
+import {
+  Button,
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+  InputProps,
+} from 'ui/components';
+import { FadeTransition, PasswordInput } from '../../components';
 
 export const EnterPassword = () => {
   const navigate = usePopupNav();
   const { isPassword, setPassword } = useStore(passwordSelector);
-  const [inputTx, setInputTxt] = useState('');
+  const [password, setPasswordValue] = useState('');
+  const [matchError, setMatchError] = useState(true);
+
+  const handleUnlock = () => {
+    void (async function () {
+      if (await isPassword(password)) {
+        setPassword(password);
+        navigate(PopupPath.INDEX);
+      } else {
+        setMatchError(false);
+      }
+    })();
+  };
+
+  const handleChangePassword: InputProps['onChange'] = e => {
+    setPasswordValue(e.target.value);
+    setMatchError(true);
+  };
+
+  const gotoRestorePage = () =>
+    void (async function () {
+      await chrome.tabs.create({
+        url: `${chrome.runtime.getURL('page.html')}`,
+      });
+    })();
+
   return (
-    <div>
-      <form
-        onSubmit={e => {
-          e.preventDefault();
-          void (async function () {
-            if (await isPassword(inputTx)) {
-              setPassword(inputTx);
-              navigate(PopupPath.INDEX);
-            }
-          })();
-        }}
-      >
-        <label>
-          Enter your password
-          <input
-            type='text'
-            value={inputTx}
-            onChange={e => {
-              setInputTxt(e.target.value);
+    <FadeTransition>
+      <Card className='w-[300px] p-2' gradient>
```

Comment:
The popup is so small, we probably should not use cards unless it's an overlay perhaps.

I wonder if we should make the entire background the card gradient color 🤔